setup.py

```diff
@@ -9,7 +9,7 @@
 
 setuptools.setup(
     name="mkdocs-plugin-progress",
-    version="1.1.4",
+    version="1.1.5",
```

Comment:
Let’s do 1.2.0 since this is a feature release.

Reply:
👍